sqlmesh/cli/main.py

```diff
@@ -40,16 +40,19 @@ def _sqlmesh_version() -> str:
     "--gateway",
     type=str,
     help="The name of the gateway.",
+    envvar="SQLMESH_GATEWAY",
 )
 @click.option(
     "--ignore-warnings",
     is_flag=True,
     help="Ignore warnings.",
+    envvar="SQLMESH_IGNORE_WARNINGS",
 )
 @click.option(
     "--debug",
     is_flag=True,
     help="Enable debug mode.",
+    envvar="SQLMESH_DEBUG",
```

Comment:
Just a heads up that we already check it here: https://github.com/TobikoData/sqlmesh/blob/main/sqlmesh/utils/__init__.py#L218

Reply:
Ok so that one isn't needed here, will unset :)